Sources/AdMobUI/AnchoredAd.swift

```diff
@@ -30,6 +30,12 @@ public struct AnchoredAd: View {
         self.width = size.width
     }
     
+    public init(ad: Advertisement, height: CGFloat = 320, width: CGFloat = 320) {
```

Comment:
I also needed to add this - without it, XCode complained that the initializer was private - I think due to `private var showingAds`.